apps/dappstore/components/layouts/HomeLayout.tsx

```diff
@@ -0,0 +1,29 @@
+import { type FC, type ReactNode } from 'react';
+import { ScrollView } from 'react-native';
+import { View } from '@walless/gui';
+import { sharedStyles } from 'utils/style';
+
+import { Footer, Header } from '.';
+
+interface Props {
+	children: ReactNode;
+}
+
+export const HomeLayout: FC<Props> = ({ children }) => {
+	return (
+		<View style={sharedStyles.minScreen}>
+			<View style={sharedStyles.container}>
+				<Header />
+			</View>
+			<ScrollView
+				contentContainerStyle={sharedStyles.contentContainer}
+				showsVerticalScrollIndicator={false}
```

Comment:
style the scroll indicator instead of hiding it

Reply:
I have contacted Ms. Ngoc for the new UI. Will update this soon